Blip.HttpClient/Services/EventTrack/EventTrackService.cs

```diff
@@ -2,17 +2,25 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Blip.HttpClient.Exceptions;
 using Blip.HttpClient.Factories;
 using Lime.Messaging.Resources;
 using Lime.Protocol;
 using Serilog;
 using Take.Blip.Client;
 using Take.Blip.Client.Extensions.EventTracker;
+using Takenet.Iris.Messaging.Resources;
+using Takenet.Iris.Messaging.Resources.Analytics;
 
-namespace Blip.HttpClient.Services.EventTrack
+namespace Blip.HttpClient.Services.EventTracker
```

Comment:
Since you changed the `namespace`, you should change the Folder name to `EventTracker` as well

Reply:
Solved on new commit